src/Objects/MediaType.php

```diff
@@ -119,6 +120,12 @@ public static function xml(string $objectId = null): self
             ->mediaType(static::MEDIA_TYPE_TEXT_XML);
     }
 
+    public static function formUrlEncoded(string $objectId = null): self
```

Comment:
Nice! 🙌 Could we have a DocBlock added here?

Reply:
Added